CHANGELOG.md

```diff
@@ -6,6 +6,187 @@
 
 ## Improvements
 
+# v1.27.0-rc1 / 2024-04-30
+
+This is the first release candidate of the upcoming optional release of Lotus v1.27.0. This feature release includes numerous improvements and enhancements for node operators, ETH RPC-providers and storage providers. This feature release introduces Curio in Beta release.
```

Comment:
would be great if we can have a section for ETH RPC users (indexers) & list the PR included in this release they would want to check out . not a blocker for RC - but should add for final release.

Reply:
https://github.com/filecoin-project/lotus/pull/11477 specifically for eth

Not just eth, though

https://github.com/filecoin-project/lotus/pull/11934 is targeting the new actor events RPC APIs

Reply:
other eth:

 - fix: api: return the correct block gas limit in the EthAPI (#11747) ([filecoin-project/lotus#11747](https://github.com/filecoin-project/lotus/pull/11747))
 - fix: Eth API: accept input data in call arguments under field 'input' (#11505) ([filecoin-project/lotus#11505](https://github.com/filecoin-project/lotus/pull/11505))
- fix: api: Length check the array sent to eth_feeHistory RPC (#11696) ([filecoin-project/lotus#11696](https://github.com/filecoin-project/lotus/pull/11696))
- fix: api: fix EthSubscribe tipsets off by one (#11858) ([filecoin-project/lotus#11858](https://github.com/filecoin-project/lotus/pull/11858))